src/components/card_component/Card/Card.js

```diff
@@ -27,15 +27,19 @@ function Card({ characterName, house, picture }) {
 
   return (
     <section className="card">
-      <h2 className="card_subtitle">{characterName}</h2>
-      <img
-        className="character_picture"
-        src={picture}
-        alt={`Picture of ${characterName}`}
-        // Eigentlich brauchen wir 'Picture of' nicht, weil der Screenreader automatisch erkennt, dass es sich um ein Picture handelt.
-      />
-      <p className={houseColor}> House {house}</p>
-      <button>more</button>
+      <div className="picture">
+        <img
+          className="character_picture"
+          src={picture}
+          alt={`Picture of ${characterName}`}
```

Comment:
Suggestion would be to just leave it empty 😉 
```
alt=""
```

Reply:
Oh, true, that would work for the screenreader :sweat_smile:  But what is happening, when the picture could not successfully be loaded? Wouldn't the content of the alt attribute pop up, so that the user gets at least information of the picture? 🤔

Reply:
Well, yes and no. The question we have to ask ourselves is: _Is this image vital for the user to understand the content of the page?_
Here I would say it is not, as the content of this card is information about a character - with a character image. The image is purely decorative. And if the image doesn't load, we won't need to know that it is an image of the character.
Does that make sense?